windows/regf.ksy

```diff
@@ -3,7 +3,6 @@ meta:
   title: Windows registry database
   application: Windows NT and later
   xref:
-    forensicswiki: Windows_NT_Registry_File_(REGF)
```

Comment:
?????

Forensicswiki is not dead, it just has moved to another domain: https://forensicswiki.xyz/wiki/index.php?title=Windows_NT_Registry_File_(REGF)